core/pom.xml

```diff
@@ -335,7 +343,14 @@
     </dependency>
     <!-- END: Test Dependencies -->
   </dependencies>
-
+  
+  <repositories>
+    <repository>
+      <id>librdfa-rdf4j</id>
+      <url>https://raw.github.com/JulioCCBUcuenca/librdfa-java/repository/</url>
```

Comment:
This is a bit hacky... what is meant to reside at this URL?

Reply:
Since we haven't published the librdfa-rdf4j parser in maven central, I added the jars there. I added that because if while compiling librdfa is not installed, the build will fail. In that URL lives the jars of librdfa-rdf4j.

Reply:
Currently, librdfa-rdf4j is a standalone module that is not part of any23. Thus, at compiling time, any23 doesn't know about librdfa-rdf4j. If we integrate this with any, we need to deal with the requirement that librdfa-rdfa needs librdfa.

Reply:
librdfa-rdf4j should definitely be a separate module of Any23, please make sure that it is and that the require JAR dependencies are located at build and compile/runtime such.